concordia/views.py

```diff
@@ -804,7 +806,8 @@ def add_transcription_status_summary_to_projects(self, projects):
             project.transcription_statuses
 
 
-reserve_rate = lambda g, r: None if r.user.is_authenticated else "1/m"
+def reserve_rate(g, r):
+    return None if r.user.is_authenticated else "12/m"
```

Comment:
The concern I have here is that we're basically saying 12 anonymous users can work at a time. That number seems low but I'm not sure how much higher we want to make it before it's not worth having the rate-limit on this view at all.

Reply:
Doesn't the key='ip' bit mean this rate is 12/m per ip? Maybe I read the docs wrong.

Reply:
Ah yes. Flake8 didn't like the lambda functions, and black puts those def's a couple lines above the @ratelimit decorator. Not the most beautiful thing in the world. Should I wait to merge this until we have a chance to integrate with registration and login, and do the HTTP 429 thing?